src/components/battles/selectors/OpponentSelector.ts

```diff
@@ -29,7 +29,12 @@ export class OpponentSelector<TGameStartr extends FullScreenPokemon> extends Com
                 return actor.statistics.health.current !== 0;
             })
             [0] as IPokemon | undefined;
-
+        const teamName: "player" | "opponent" = Team[team] as "player" | "opponent";
+        if (teamName === "opponent") {
+            const pkmn: IPokemon = this.gameStarter.itemsHolder.getItem("PokemonInParty")[0];
```

Comment:
Nit: name it `pokemon` instead of `pkmn`. It's easier to understand at first glance.